tests/conftest.py

```
@@ -0,0 +1,29 @@
+"""Shared pytest fixtures for Animated Scenes integration tests."""
+
+import pytest
+
+from custom_components.animated_scenes.const import DOMAIN
+
+pytest_plugins = ("pytest_homeassistant_custom_component",)
+
+
+@pytest.fixture(autouse=True)
+def auto_enable_custom_integrations(enable_custom_integrations: None) -> None:
```

Ruff (0.15.15)

[warning] Unused function argument: `enable_custom_integrations`

(ARG001)